app/uk/gov/hmrc/alcoholdutyreturns/controllers/ReturnsController.scala

```diff
@@ -51,6 +53,22 @@ class ReturnsController @Inject() (
         )
     }
 
+  def submitReturn(appaId: String, periodKey: String): Action[JsValue] =
+    authorise(parse.json).async { implicit request =>
+      withJsonBody[AdrReturnSubmission] { returnSubmission =>
+        returnsService
+          .submitReturn(returnSubmission, ReturnId(appaId, periodKey))
```

Comment:
Should validate that if a section is not filled in, the items are not present (Seq/Set being empty)

Reply:
Validation will happen in a subsequent jira.